capstone/src/main/webapp/youtubeScript.js

```diff
@@ -186,31 +204,27 @@ function showAvailableAttributes() {
     label.appendChild(document.createTextNode(attribute));
     avaiableAttributesElement.appendChild(checkbox);
     avaiableAttributesElement.appendChild(label);
-    avaiableAttributesElement.appendChild(document.createTextNode (" "));
+    avaiableAttributesElement.appendChild(document.createTextNode(" "));
```

Comment:
nit: `avaiableAttributesElement` -> `availableAttributesElement`

Reply:
addressed in "fix spelling"